make/common/JarArchive.gmk

```diff
@@ -193,7 +193,8 @@ define SetupJarArchiveBody
   $1_UPDATE_CONTENTS=\
       if [ "`$(WC) -l $$($1_BIN)/_the.$$($1_JARNAME)_contents | $(AWK) '{ print $$$$1 }'`" -gt "0" ]; then \
         $(ECHO) "  updating" `$(WC) -l $$($1_BIN)/_the.$$($1_JARNAME)_contents | $(AWK) '{ print $$$$1 }'` files && \
-        $$($1_JAR_CMD) --update $$($1_JAR_OPTIONS) --file $$@ @$$($1_BIN)/_the.$$($1_JARNAME)_contents; \
+        $(CAT) $$($1_BIN)/_the.$$($1_JARNAME)_contents | $(SORT) > $$($1_BIN)/_the.$$($1_JARNAME)_contents_sorted && \
```

Comment:
I'm very grateful that you found this and fixed it, but this code could really hand you a ["Useless Use of Cat Award"](https://porkmail.org/era/unix/award#cat). :-D

`sort in-file > out-file` is much better.

Reply:
I like to make my mark :-)
i'm just so used to piping !

Reply:
Fixed